packages/cli/__tests__/zosfiles/__unit__/create/ds/ds.handler.unit.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Create, CreateDataSetTypeEnum } from "@zowe/zos-files-for-zowe-sdk";
+import { UNIT_TEST_ZOSMF_PROF_OPTS } from "../../../../../../../__tests__/__src__/mocks/ZosmfProfileMock";
+
+describe("Create data set handler", () => {
+    describe("process method", () => {
+        it("should create a  data set if requested", async () => {
+            // Require the handler and create a new instance
+            const handlerReq = require("../../../../../src/zosfiles/create/ds/ds.handler");
+            const handler = new handlerReq.default();
+            const dataSetName = "testing";
+            const likeDataSetName = "testing";
+
+            // Vars populated by the mocked function
+            let error;
+            let apiMessage = "";
+            let jsonObj;
+            let logMessage = "";
+            let AbstractSession = null;
+
+            // Mock the create function
+            Create.dataSetLike = jest.fn((session) => {
+                AbstractSession = session;
+                return {
+                    success: true,
+                    commandResponse: "created"
+                };
+            });
+
+            // Mocked function references
+            const profFunc = jest.fn((args) => {
+                return {
+                    host: "fake",
+                    port: "fake",
+                    user: "fake",
+                    password: "fake",
+                    auth: "fake",
+                    rejectUnauthorized: "fake"
+                };
+            });
+
+            try {
+                // Invoke the handler with a full set of mocked arguments and response functions
+                await handler.process({
+                    arguments: {
+                        $0: "abstract",
+                        _: ["abstract"],
+                        dataSetName,
+                        likeDataSetName,
```

Comment:
This is shorthand for `likeDataSetName: likeDataSetName`. Since the command was reworked to have a "--like" argument instead of a "likeDataSetName" positional, it needs to be called "like" instead. Please change to `like: likeDataSetName`